### DISCUSSION TITLE
Form close loop between MatterSim and MatterGen

### DISCUSSION THREAD

**Discussion by yuhao-morning** [microsoft/mattergen#147] [Ideas]
I have a question that I come up with when I recently use Mattersim and MatterGen. I see that MatterGen takes 101 number of atoms totally, but MatterSim takes only 95 number of atoms totally. I find this by printing the logits of generated samples by d3pm_predictor_corrector in MatterGen, and the atom_embedding layer shape of MatterSim m3gnet architecture. Could you tell me which atoms these two models use, because we know that in periodic table of elements there are totally 128 atoms, larger than these two.  So for me, I can't directly use the generated samples of MatterGen (after changing the format) as the input to mattersim to predict its properties, failing to form a close loop.  So could you give me suggestions or other ideas on how I can achieve this, i.e. use MatterSim to predict the properties of generated samples of MatterGen ?

**Comment by ClaudioZeni** [comment]
The set of elements supported by MatterGen and MatterSim differs slightly, with the former supporting a more restricted set of elements.
Nonetheless, any structure generated by MatterGen can be simulated using MatterSim once it has been saved in an appropriate format. Commonly used formats include Python classes such as `Ase.Atoms` and `Pymatgen.core.Structure`.
